packages/ui/components/form/inputs/Input.tsx

```diff
@@ -61,13 +60,15 @@ type AddonProps = {
   isFilled?: boolean;
   className?: string;
   error?: boolean;
-  paddingX?: number;
 };
 
-const Addon = ({ isFilled, children, className, error, paddingX = 12 }: AddonProps) => (
```

Comment:
Is the default padding lost?

Reply:
Was covered with `px-3` here: https://github.com/calcom/cal.com/pull/5704/files#diff-81bcd4a4451f9e94b8bf86f92909396dd2dd9dd0598ae8ff185b7d0a29f9320cR68